src/App.js

```diff
@@ -1,38 +1,36 @@
-import InputSearch from "./react/componets/InputSearch/InputSearch";
-import Header from "./react/componets/Header/Header";
-import "./App.css";
-import { TaskList } from "./react/componets/TaskLIist/TaskList";
-import { TaskListContextProvider } from "./react/context/TaskListContext";
+// import InputSearch from "./react/componets/InputSearch/InputSearch";
+// import Header from "./react/componets/Header/Header";
+// import "./App.css";
+// import { TaskList } from "./react/componets/TaskLIist/TaskList";
+// import { TaskListContextProvider } from "./react/context/TaskListContext";
 
- function App() {
-   return (
-     <TaskListContextProvider>
-       <Header />
-       <InputSearch />
-       <TaskList />
-     </TaskListContextProvider>
-   );
- }
+//  function App() {
+//    return (
+//      <TaskListContextProvider>
+//        <Header />
+//        <InputSearch />
+//        <TaskList />
+//      </TaskListContextProvider>
+//    );
+//  }
 
- export default App;
+//  export default App;
 
-// import InputSearch from "../src/redux/componets/InputSearch/InputSearch";
-// import Header from "../src/redux/componets/Header/Header";
-// import "./App.css";
-// import { TaskList } from "../src/redux/componets/TaskList/TaskLisr";
-// import { Provider } from "react-redux";
-// import rootReducer from "../src/redux/store/index";
-// import { configureStore } from "@reduxjs/toolkit";
+import InputSearch from "../src/redux/componets/InputSearch/InputSearch";
+import Header from "../src/redux/componets/Header/Header";
+import "./App.css";
+import { TaskList } from "../src/redux/componets/TaskList/TaskLisr";
+import { Provider } from "react-redux";
```

Comment:
It will beter to put any global `import`s to top of the file, and after them our local `import`s
for example:
`import React`
`import react-redux`
`import Button from MUI`
`import MyComponent`
`import MyStyles'